app/models/ride.rb

```diff
@@ -6,4 +6,24 @@ class Ride < ActiveRecord::Base
   validates :seat_count, presence: true
   validates :date, presence: true
   validates :departure_time, presence: true
+
+  def full?
+    passengers.count >= seat_count
```

Comment:
You could use either `passenger_count` or replace the body with `seats_available > 0`